johnsnowlabs/py_models/jsl_secrets.py

```diff
@@ -266,6 +264,12 @@ def build_or_try_find_secrets(
             if not secrets and not force_browser:
                 # Search Env Vars
                 secrets = JslSecrets.search_env_vars()
+            
+            if settings.enforce_versions and not JslSecrets.is_hc_secret_correct_version(secrets.HC_SECRET) and not JslSecrets.is_ocr_secret_correct_version(secrets.OCR_SECRET):
+                # Make sure secrets and versions re enforced
+                print("👷 Trying to install compatible secrets. Use nlp.settings.enforce_versions=False if you want to install outdated secrets.")
+                JslSecrets.enforce_versions(secrets)
+
```

Comment:
You want to update the old secrete variable with the new one
`secrets = JslSecrets.enforce_versions(secrets)`

Reply:
Done